### ISSUE TITLE
Borderlands 2 (team_Utils::GetAppID)

### ISSUE THREAD

**Issue by cccpvs** [otavepto/gbe_fork#120]
**API**: release-sdk_159-overlay_font(hotfix_1)
**LOG**: [STEAM_LOG.txt](https://github.com/otavepto/gbe_fork/files/14894778/STEAM_LOG.txt)

**steam_settings** is made via **generate_emu_config**.
The application closes immediately on startup.
Also tried **experimental** and **experimental_steamclient** no effect too.
[COLD_LDR_LOG.txt](https://github.com/otavepto/gbe_fork/files/14894801/COLD_LDR_LOG.txt)

```
steam_settings\achievements.json
steam_settings\build_id.txt
steam_settings\depots.txt
steam_settings\disable_account_avatar.txt
steam_settings\disable_lan_only.txt
steam_settings\disable_networking.txt
steam_settings\disable_overlay.txt
steam_settings\disable_overlay_achievement_notification.txt
steam_settings\disable_overlay_friend_notification.txt
steam_settings\disable_source_query.txt
steam_settings\DLC.txt
steam_settings\steam_appid.txt
steam_settings\steam_interfaces.txt
steam_settings\supported_languages.txt
steam_settings\img\...
```

**Comment by cccpvs** [comment]
Yeah, I was wrong about the **steam_appid**.
I replaced everything I needed and generated a new **steam_settings** 
Now the game started, but almost immediately an error came out.
Here is the error and the log.
### Normal api
![Clipboard 1](https://github.com/otavepto/gbe_fork/assets/93628981/24e50cf6-6f0e-4ca5-a22a-6bfbeaf37d23)
[STEAM_LOG.txt](https://github.com/otavepto/gbe_fork/files/14898217/STEAM_LOG.txt)

And the experimental version with **steamclient**.dll also shows a similar error.
[STEAM_LOG.txt](https://github.com/otavepto/gbe_fork/files/14898230/STEAM_LOG.txt)

### ColdClientLoader
Used a new AppId already and there was no effect.
[COLD_LDR_LOG.txt](https://github.com/otavepto/gbe_fork/files/14898236/COLD_LDR_LOG.txt)
After that I used it with DllsToInjectFolder=extra_dlls
where I didn't get any effect either. The game did not start.
[COLD_LDR_LOG.txt](https://github.com/otavepto/gbe_fork/files/14898245/COLD_LDR_LOG.txt)
I also tried different ways of launching through **ColdClientLoader.ini.** No result either.

**Comment by cccpvs** [comment]
Unfortunately at once and did not notice that there is no file steam_interfaces.txt, which apparently was not created through **generate_emu_config-win**, I created it through find_interfaces and the game **loaded** as needed and without errors. 
[STEAM_LOG.txt](https://github.com/otavepto/gbe_fork/files/14898353/STEAM_LOG.txt)
I think the matter is closed. I'll let you know what happens.

**Comment by cccpvs** [comment]
File **STEAM_LOG.txt** is not created only if you specify` ForceInjectSteamClient=1` in the parameter.
And then its content is too short

```
[0 ms, 210 us] [tid 8360] Hooking lan only functions
[0 ms, 692 us] [tid 8360] Crack file D:\SteamLibrary\steamapps\common\Borderlands_2_RU\Binaries\Win32\cracksteam_api.dll
```

**Comment by cccpvs** [comment]
Yes at every launch appears a file like **Borderlands2.exe.****.STEAMSTART**.

### die.exe
![Clipboard 2](https://github.com/otavepto/gbe_fork/assets/93628981/2d1b90a2-31cc-454c-9508-8a88fb3baecc)

### diel.exe
```
MSDOS
    Operation system: MS-DOS[8086, 16-bit, EXE]
PE32
    Operation system: Windows(XP)[I386, 32-bit, GUI]
    Linker: Microsoft Linker(10.0)
    Compiler: EP:Microsoft Visual C/C++(2008-2010)[EXE32]
    Compiler: Microsoft Visual C/C++(15.00-16.00)[msvcrt]
    Language: C/C++
    Tool: Visual Studio
    Sign tool: Windows Authenticode(2.0)[PKCS #7]
    Protection: Generic(Heuristic)[Custom DOS]
    Packer: Packer detected(Heuristic)[Section 5 (".rsrc") compressed]
    Overlay: Binary
        Certificate: WinAuth(2.0)[PKCS #7]
```
I hope I did what I needed to do.

**Comment by LuKeSt0rm** [comment]
game is protected with CEG and you must first remove/emulate the CEG protection. 
You can do this with LumaCEG together with the original game on steam. (I won't describe more details here)
